buildspec.yml

```diff
@@ -0,0 +1,16 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 10
```

Comment:
Why are we using this version? LTS is 12 now. Can't we use a flag for LTS instead of writing explicity the version number?